lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -0,0 +1,74 @@
+require 'gds_api/test_helpers/json_client_helper'
+require 'gds_api/test_helpers/content_item_helpers'
+require 'gds_api/test_helpers/intent_helpers'
+require 'json'
+
+module GdsApi
+  module TestHelpers
+    module PublishingApiV2
+      include ContentItemHelpers
+
+      PUBLISHING_API_V2_ENDPOINT = Plek.current.find('publishing-api') + '/v2'
+
+      def stub_publishing_api_v2_put_content(content_id, body)
+        stub_publishing_api_v2_put(content_id, body, '/content')
+      end
+
+      def stub_publishing_api_v2_put_links(content_id, body)
+        stub_publishing_api_v2_put(content_id, body, '/links')
+      end
+
+      def stub_publishing_api_v2_publish(content_id, body)
+        url = PUBLISHING_API_V2_ENDPOINT + "/content/#{content_id}/publish"
+        stub_request(:post, url).with(body: body).to_return(status: 200, body: '{}', headers: {"Content-Type" => "application/json; charset=utf-8"})
+      end
+
+      def stub_default_publishing_api_v2_put()
```

Comment:
I think a much better name for this would be `stub_any_publishing_api_v2_put_content`. If we want it to stub PUTs to links (which this doesn't do currently), then I suppose it should lose my proposed `_content`.